test/controllers/api/audio_files_controller_test.rb

```diff
@@ -16,6 +16,7 @@
         upload: 'http://thisisatest.com/guid1/test.mp3',
         set_account_uri: api_account_url(story.account)
       }
+      @request.env["CONTENT_TYPE"] = "application/json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.